source/fleetprovisioning/FleetProvisioning.cpp

```diff
@@ -551,6 +555,34 @@ bool FleetProvisioning::ProvisionDevice(shared_ptr<SharedCrtResourceManager> fpC
  * Helper methods
  */
 
+bool FleetProvisioning::LocateDeviceKey(const string filePath)
+{
+    struct stat info;
+    bool locatedDeviceKey = true;
+    string expandedPath = FileUtils::ExtractExpandedPath(filePath.c_str());
+    if (stat(expandedPath.c_str(), &info) != 0)
+    {
+        LOGM_ERROR(
+            TAG,
+            "*** %s: Failed to find the device private key %s, file does not exist ***",
+            DeviceClient::DC_FATAL_ERROR,
```

Comment:
Is this actually a fatal error? I thought that if we can't find the private key, we are supposed to default back to the other FP API?

Reply:
My bad. I'll update the log.